x-pack/plugins/security_solution/public/common/components/event_details/table/add_to_timeline_cell.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { isEmpty } from 'lodash';
+
+import { useKibana } from '../../../lib/kibana';
+import { AlertSummaryRow } from '../helpers';
+import { useActionCellDataProvider } from './use_action_cell_data_provider';
+
+const AddToTimelineCell = React.memo<AlertSummaryRow['description']>(
```

Comment:
Please add the unit test here

Reply:
This already has one `add_to_timeline_cell.test.tsx`